src/solvers/refinement/string_constraint_generator_main.cpp

```diff
@@ -576,7 +576,7 @@ std::pair<exprt, string_constraintst> add_axioms_for_char_at(
   array_string_exprt str = get_string_expr(array_pool, f.arguments()[0]);
   symbol_exprt char_sym = fresh_symbol("char", str.type().subtype());
   const exprt constraint = equal_exprt(char_sym, str[f.arguments()[1]]);
-  return {char_sym, {{constraint}}};
+  return {char_sym, {{constraint}, {}, {}}};
```

Comment:
This seems ok, but maybe a clearer solution would be to declare the string_constrainst explicitely:
```
string_constraintst constraints;
constraints.existential = equal_exprt(char_sym, str[f.arguments()[1]]);
return {char_sym, constraints};
```
to replace the last two lines.